FLT/EllipticCurve/Torsion.lean

```diff
@@ -102,8 +102,11 @@ noncomputable instance WeierstrassCurve.galoisRepresentation_smul
 noncomputable def WeierstrassCurve.galoisRepresentation
     (K : Type u) [Field K] [DecidableEq K] [Algebra k K] :
     DistribMulAction (K ≃ₐ[k] K) (E ⟮K⟯) where
-      one_smul := sorry -- these should all be easy
-      mul_smul := sorry
+      one_smul := fun P ↦ by
+        change WeierstrassCurve.Points.map E (AlgHom.id k K) P = P
+        rw [WeierstrassCurve.Points.map_id E K]
+        rfl
```

Comment:
Yes it works, but what really needs to be done here is that we need to change `WeierstrassCurve.Points.map` to `WeierstrassCurve.Affine.Point.map` now mathlib has it (it didn't have it when I wrote this code originally) and adapt to the new mathlib spelling of everything, rather than going ahead building more stuff which will need to be refactored.